src/Footer.jsx

```diff
@@ -52,7 +52,15 @@ const Footer = () => {
                     <img src={nostr} alt="Nostr Logo" />
                 </a>
             </div>
-            <p>{t("footer")}</p>
+            <p>
+                {t("footer")}
+                <br />
+                <a
+                    target="_blank"
+                    href="https://amboss.space/node/026165850492521f4ac8abd9bd8088123446d126f648ca35e60f88177dc149ceb2">
+                    {t("lightningnode")}
+                </a>
+            </p>
```

Comment:
Still don't like that link here

Reply:
Me neither, it belongs as title for the node stats on the front page, see https://github.com/BoltzExchange/boltz-web-app/pull/140#pullrequestreview-1491070483